analytical/templatetags/yandex_metrica.py

```diff
@@ -12,25 +12,15 @@
 
 COUNTER_ID_RE = re.compile(r'^\d{8}$')
```

Comment:
Another user reported in #244 that the counter id can now be longer that 8 characters. Can you also adjust this regular expression accordingly?

```suggestion
COUNTER_ID_RE = re.compile(r'^\d{8,10}$')
```